backstage/values.yaml

```diff
@@ -149,6 +153,8 @@ appConfig:
     baseUrl: https://demo.example.com/lighthouse-api
   rollbar:
     organization: roadie
+  jiraProxy:
+    enabled: false
```

Comment:
btw, we might want to have this enabled by default

Reply:
:+1: enabled by default